tasks/kernel_matrix_testing/README.md

```diff
@@ -0,0 +1,231 @@
+# Kernel Matrix Testing system
+
+## Overview
+The Kernel Matrix Testing system is a new approach for testing system-probe. It uses libvirt and qemu to launch pre-provisioned VMs over a range of distributions and kernel versions. These VMs are used for running the test suite of system-probe.  
+
+Developers can check out this confluence page for more details about the system.   
+
+This file will document the invoke tasks provided to easily manage the lifecycle of the VMs launched using this system.
+
+The system works on the concept of `stacks`. A `stack` is a collection of VMs, both local and remote. A `stack` is given a unique name by the user. Convenience options for generating the name of the `stack` from the current branch is also provided. This allows the developers to couple `stacks` with their git workflow.
+A `stack` may be:
+- Created
+- Configured
+- Launched
+- Paused
+- Resumed
+- Destroyed
+
+> All subsequent commands are assumed to be executed from the root directory of the datadog-agent repository.
+
+## Dependencies
+### Linux
+Review and run `tasks/kernel_matrix_testing/env-setup.sh`   
+   
+Download [test-infra-definitions](https://github.com/DataDog/test-infra-definitions) repository.   
+From within the repository execute the following commands:  
+```bash
+go mod download
+export PULUMI_CONFIG_PASSPHRASE=dummy
+pulumi --non-interactive plugin install
+pulumi --non-interactive plugin ls
+```
+
+### MacOS
```

Comment:
I think we should mention that mac users can use this framework with bare metal machine instance in aws.

later in the readme (in `Initializing the environment` section) you mention mac users as they can use the framework, while from section it sounds like mac users can't do that)